### PR TITLE
Css clean up

### PR DESCRIPTION
**Pull request by fyun89**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env", "react"]
+}
```
Comment (etcaples) [inline .babelrc]: newline

File: .eslintrc.js [hunk 2]
```diff
@@ -1,10 +1,3 @@
...
- * These rules enforce the Hack Reactor Style Guide
- *
- * Visit this repo for more information:
- *   https://github.com/reactorcore/eslint-config-hackreactor
- */
-
 module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
+    "extends": "airbnb"
 };
```
Comment (etcaples) [inline .eslintrc.js]: newline

File: client/dist/styles.css [hunk 3]
```diff
@@ -117,8 +125,9 @@ hr{
   background-size: 100%;
   left: 0px;
   top: 130px;
-}
+} */
 
+/* Scroll.jsx */
 .previous-arrow:hover{
   display:block;
   position: absolute;
```
Comment (etcaples) [inline client/dist/styles.css]: all the code above that's commented out - do you need those, or can they be deleted?

File: client/src/components/App.jsx [hunk 4]
```diff
@@ -1,5 +1,141 @@
...
+#quietPrice{
+  border - right: 1 lightgray;
+  display: inline;
+  color: lightgray;
+  font - family: 'swansea';
+  font - size: 12px;
+}
+
+#listEntry: hover #image0{
+
```
Comment (etcaples) [inline client/src/components/App.jsx]: do you need this empty object?

File: client/src/components/App.jsx [hunk 5]
```diff
@@ -1,5 +1,141 @@
...
+  font - size: 12px;
+  font - family: 'swansea';
+}
+
+#quietPrice{
+  border - right: 1 lightgray;
+  display: inline;
+  color: lightgray;
+  font - family: 'swansea';
+  font - size: 12px;
```
Comment (etcaples) [inline client/src/components/App.jsx]: are there supposed to be spaces around the hyphens?

File: client/src/components/CarouselList.jsx [hunk 6]
```diff
@@ -65,12 +63,12 @@ class CarouselList extends React.Component {
       marginRight: '50px',
     };
 
-    var subcontainerStyle = {
+    let subcontainerStyle = {
```
Comment (etcaples) [inline client/src/components/CarouselList.jsx]: do you ever reassign this? could it be a const?

File: client/src/components/CarouselListEntries.jsx [hunk 7]
```diff
@@ -11,63 +21,107 @@ class CarouselListEntries extends React.Component {
...
   }
+ 
 
   componentDidMount() {
-    var currentCarouselItem = this.props.product;
-    var imageLinks = [];
-    var productInfo = null;
+    let currentCarouselItem = this.props.product;
+    const imageLinks = [];
+    let productInfo = null;
```
Comment (etcaples) [inline client/src/components/CarouselListEntries.jsx]: do you ever reassign productInfo, or could it be const?

File: client/src/components/CarouselListEntries.jsx [hunk 8]
```diff
@@ -11,63 +21,107 @@ class CarouselListEntries extends React.Component {
       product: null,
     };
   }
+ 
 
   componentDidMount() {
-    var currentCarouselItem = this.props.product;
-    var imageLinks = [];
-    var productInfo = null;
+    let currentCarouselItem = this.props.product;
```
Comment (etcaples) [inline client/src/components/CarouselListEntries.jsx]: do you ever reassign this variable, or could it be const?

File: client/src/components/CarouselListEntries.jsx [hunk 9]
```diff
@@ -11,63 +21,107 @@ class CarouselListEntries extends React.Component {
...
           }
         </div>
-      </div>
+      </ListEntry>
     );
   }
 }
 
-export default CarouselListEntries;
+export default CarouselListEntries;
```
Comment (etcaples) [inline client/src/components/CarouselListEntries.jsx]: newline

File: client/src/components/Heart.jsx [hunk 10]
```diff
@@ -6,7 +6,7 @@ class Heart extends React.Component {
     super(props);
     this.state = {
       heart: this.props.myHeart,
-      style: "./sprites/002-unheart.svg"
+      style: './sprites/002-unheart.svg'
```
Comment (etcaples) [inline client/src/components/Heart.jsx]: missing comma after this svg styles

File: client/src/components/Heart.jsx [hunk 11]
```diff
@@ -15,12 +15,12 @@ class Heart extends React.Component {
     if (this.props.myHeart === 1) {
       this.setState({
         heart: this.props.myHeart,
-        style: "./sprites/001-heart.svg"
+        style: './sprites/001-heart.svg'
```
Comment (etcaples) [inline client/src/components/Heart.jsx]: missing comma after this svg styles

File: client/src/components/Heart.jsx [hunk 12]
```diff
@@ -15,12 +15,12 @@ class Heart extends React.Component {
...
       });
     } else {
       this.setState({
         heart: this.props.myHeart,
-        style: "./sprites/002-unheart.svg"
+        style: './sprites/002-unheart.svg'
```
Comment (etcaples) [inline client/src/components/Heart.jsx]: missing comma after this svg styles

File: client/src/components/Heart.jsx [hunk 13]
```diff
@@ -31,33 +31,34 @@ class Heart extends React.Component {
...
         heart: 1,
-        style: "./sprites/001-heart.svg",
+        style: './sprites/001-heart.svg',
       });
     }
   }
 
   render(props) {
-    var currentStyle = {
+    let currentStyle = {
```
Comment (etcaples) [inline client/src/components/Heart.jsx]: can this be const?

File: client/src/components/Picture.jsx [hunk 14]
```diff
@@ -1,73 +1,142 @@
...
+    //   top: '-945px',
+    //   background: `url(${image[3]})`, // changed from url(${image[3]}) to conserve GET request from S3
+    //   backgroundRepeat: 'no-repeat',
+    //   backgroundPosition: 'center',
+    //   width: '130px',
+    //   height: '420px',
+    //   backgroundSize: '100%',
+    //   marginLeft: '20px',
+    //   marginRight: '20px',
+    // };
```
Comment (etcaples) [inline client/src/components/Picture.jsx]: can you delete all these comments, or are you still using them?

File: client/src/components/Picture.jsx [hunk 15]
```diff
@@ -1,73 +1,142 @@
...
+    console.log(image)
```
Comment (etcaples) [inline client/src/components/Picture.jsx]: if you're not still using this, remove loose console.log

File: client/src/components/__tests__/enzyme-test-file.js [hunk 16]
```diff
@@ -0,0 +1,13 @@
...
+import CarouselList from '../CarouselList.jsx';
+import { shallow, render, mount } from 'enzyme';
+
+describe('Render', () => {
+  it('should have state.currentProduct set as null', () => {
+    const wrapper = shallow(<Carousel />)
+    const stateExists = wrapper.state().currentProduct;
+    expect(stateExists).toBe(null)
+  });
+});
```
Comment (etcaples) [inline client/src/components/__tests__/enzyme-test-file.js]: newline

File: jest.config.js [hunk 17]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  'setupFiles': ['<rootDir>/setup.js'],
+  'testEnvironment': 'node',
+  'globals': {
+    'window': true,
+  }
+}
```
Comment (etcaples) [inline jest.config.js]: newline

File: client/dist/styles.css (hunk 3 above)
Comment (fyun89) [reply client/dist/styles.css]: I might need these for future refactoring. Thanks for pointing out tho!

File: client/src/components/Picture.jsx (hunk 14 above)
Comment (fyun89) [reply client/src/components/Picture.jsx]: Yeah I decided to leave them for future reference. Thanks!

File: client/src/components/App.jsx (hunk 4 above)
Comment (fyun89) [reply client/src/components/App.jsx]: Oops. Didn't need that! Thanks!

**Pull request merged**